src/message.rs

```diff
@@ -64,3 +65,12 @@ impl Message {
         }
     }
 }
+
+/// Style of output we should produce
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
+pub enum OutputFormat {
```

Comment:
A pedantic detail in case relevant: OutputFormat derives ValueEnum which is a clap-derive thing. As such, message.rs now depends on clap. That could be a problem if you had dreams of a really hard library/binary split but I don't think you do, so this is fine!

Reply:
yeah i hear ya, makes sense. if/when we end up wanting that, it's an easy enough change